src/hotspot/share/gc/shenandoah/shenandoahPhaseTimings.cpp

```diff
@@ -55,10 +55,10 @@ ShenandoahPhaseTimings::ShenandoahPhaseTimings(uint max_workers) :
 
   // Initialize everything to sane defaults
   for (uint i = 0; i < _num_phases; i++) {
-#define SHENANDOAH_WORKER_DATA_NULL(type, title) \
-    _worker_data[i] = NULL;
-    SHENANDOAH_PAR_PHASE_DO(,, SHENANDOAH_WORKER_DATA_NULL)
-#undef SHENANDOAH_WORKER_DATA_NULL
+#define SHENANDOAH_WORKER_DATA_nullptr(type, title) \
+    _worker_data[i] = nullptr;
+    SHENANDOAH_PAR_PHASE_DO(,, SHENANDOAH_WORKER_DATA_nullptr)
+#undef SHENANDOAH_WORKER_DATA_nullptr
```

Comment:
Fix these macros

Reply:
My suggestion would be to leave the macro name unchanged, just change the definition. Or at least use uppercasing in the macro name. (Perhaps the latter's what your comment intended.)

More generally, is there a `jcheck` rule that will prevent re-introduction of NULL usage into the mix?

Reply:
This code diff is out of date, the macro name was reverted to the original in a subsequent commit. Good question about adding this to `jcheck`.